modules/canopennode/CO_driver.c

```diff
@@ -31,6 +31,8 @@ K_MUTEX_DEFINE(canopen_send_mutex);
 K_MUTEX_DEFINE(canopen_emcy_mutex);
 K_MUTEX_DEFINE(canopen_co_mutex);
 
+static canopen_rxmsg_callback rxmsg_callback;
```

Comment:
```suggestion
static canopen_rxmsg_callback rxmsg_callback = NULL;
```

Reply:
The compliance check does not seem to like this change. Shall I revert it?

Reply:
Yes, sorry about that.